README.md

```diff
@@ -200,4 +224,5 @@ Support this project by [donating on Gratipay](https://gratipay.com/jzaefferer/)
 
 [1]: http://contribute.jquery.org/commits-and-pull-requests/#commit-guidelines
 [2]: https://github.com/angular/angular.js/blob/master/CONTRIBUTING.md#commit
-[3]: https://github.com/typicode/husky
\ No newline at end of file
+[3]: https://github.com/typicode/husky
+[4]: bit.ly/commit-range
```

Comment:
This should include a protocol. I'd link to https://git-scm.com/book/en/v2/Git-Tools-Revision-Selection#Commit-Ranges directly.